.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
   one-var-declaration-per-line: error
   operator-linebreak: [error, after]
   quotes: [error, single, avoid-escape]
   semi: error
   semi-spacing: error
   space-before-blocks: [error, always]
   space-before-function-paren: [error, {
-    "anonymous": "never",
-    "named": "never",
-    "asyncArrow": "always"
+    anonymous: never,
```

Comment:
I agree with @aqrln in the general case, but in this case since it's most probably gonna be backported by cherry picking just the rules change, then running `eslint --fix`, hopefully this wouldn't cause a mess 🙏